Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimedAmountInputCell.swift

```diff
@@ -36,6 +38,11 @@ class ClaimedAmountInputCell: UITableViewCell {
         fieldDelegate.isUsingMaxValue
     }
 
+    var redeemDeadlineWarning: Bool {
+        get { !redeemWarningLabel.isHidden }
```

Comment:
We don't need that getter, I would remove it and use set(redeemDeadlineLabelVisible: Bool) method